### PR TITLE
Add native-py fix to buildifier

### PR DESCRIPTION
**Pull request by brandjon**
This is pretty much a copy/paste of #671 for Python rules. It migrates for the
incompatible change `--incompatible_load_python_rules_from_bzl` (bazelbuild/bazel#9006).

### CODE REVIEW

**Comment by brandjon** [comment]
Missing from this automated migration is adding `@rules_python` to the WORKSPACE. Does/should buildifier help automate that? If not, we should probably update the warnings file to document this and link to the rules_python readme (same for java and cc).

File: warn/warn.go [hunk 1]
```diff
@@ -130,6 +130,7 @@ var FileWarningMap = map[string]func(f *build.File) []*LinterFinding{
 	"native-build":              nativeInBuildFilesWarning,
 	"native-cc":                 nativeCcRulesWarning,
 	"native-java":               nativeJavaRulesWarning,
+	"native-py":                 nativePyRulesWarning,
```
Comment (vladmos) [inline warn/warn.go]: Move two lines below

Comment (brandjon) [reply warn/warn.go]: Done.

**Comment by brandjon** [comment]
Thanks!

**Pull request merged**